dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
If there is a possibility that RA8M2 or similar will appear in the future, ra8xx.dtsi would be better.

Reply:
We cannot decide it at this time. Let us keep as current and consider it when 2nd generation is available.